.github/workflows/publish.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - 'main'
+      - 'devops/add-docker-sec'
```

Comment:
not need right?

Reply:
right! removed in https://github.com/babylonlabs-io/vigilante/pull/100/commits/5440451188f763e450ddd8a207c3fecfc013862f

Reply:
looks good feel free to merge